config/jobs/kubernetes/kops/kops-periodics-versions.yaml

```diff
@@ -32,38 +32,6 @@ periodics:
     testgrid-dashboards: google-aws, sig-cluster-lifecycle-kops
     testgrid-tab-name: kops-aws-k8s-latest
 
-- interval: 2h
```

Comment:
any reason to remove this? are you thinking that its a duplicate of the k8s-latest job above? that probably makes sense.

Reply:
Yes, pretty much duplicate. I did not see a good reason for it to exist.

Reply:
ok that sounds good to me. as long as we keep at least one job testing with the latest k8s commit then we should be sufficient here 👍